### PR TITLE
Update to gleam 0.5

### PR DESCRIPTION
**Pull request by nox**
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/2717)

### CODE REVIEW

File: webrender/src/device.rs [hunk 1]
```diff
@@ -701,11 +702,19 @@ impl Device {
         _file_changed_handler: Box<FileWatcherHandler>,
         cached_programs: Option<Rc<ProgramCache>>,
     ) -> Device {
-        let max_texture_size = gl.get_integer_v(gl::MAX_TEXTURE_SIZE) as u32;
+        let mut max_texture_size = 0;
```
Comment (kvark) [inline webrender/src/device.rs]: wouldn't it be easier to just go with `let mut max_texture_size = [0]`, followed by `&mut max_texture_size` and then taking `[0]` from it as opposed to relying on external `ref_slice` crate?

Comment (nox) [reply webrender/src/device.rs]: Maybe, which way do you prefer? I don't particularly care either way, I went for the simpler code that made the less invasive changes, not having to do `[0]` everywhere.

**Comment by nox** [comment]
I don't understand what failed, and I don't understand Taskcluster's UI.

**Comment by staktrace** [comment]
Clicking on the link takes you to the task group view, which shows you the different tasks. The one that is marked failed as of right now is "Linux release tests", so you can click on that to view the task detail. Then you need to click on the "Run Logs" tag and select the log (either live.log or live_backing.log will do, they should be the same). The error is:

```
[taskcluster:error] Task was aborted because states could not be created successfully. Error: Error calling 'link' for localLiveLog : Error: (HTTP code 500) server error - connection error: desc = "transport: dial unix:///var/run/docker/containerd/docker-containerd.sock: timeout" 
```

which is some sort of intermittent network failure. We ran into this yesterday as well. Waiting a bit and retrying will probably resolve it

**Comment by nox** [comment]
@staktrace Oh thanks! Will retry in a bit then.

**Pull request merged**